pkg/network/protocols/amqp/server.go

```diff
@@ -0,0 +1,38 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package amqp
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/network/protocols/http/testutil"
+	"github.com/stretchr/testify/require"
```

Comment:
nit:

```suggestion
	"github.com/stretchr/testify/require"

	"github.com/DataDog/datadog-agent/pkg/network/protocols/http/testutil"
```

Reply:
when I change it to the suggest order it marks it as warning, then I do the suggested change it it returns to the shape I committed.